### DISCUSSION TITLE
Issue setting Vercel nameservers

### DISCUSSION THREAD

**Discussion by Thunear** [vercel/community#4386] [Help] [answered]
### Summary

Hi there! We have a Norwegian domain (designsystemet.no), and we want to move our DNS to Vercel so we can use it as a wildcard domain. We aren't able to set the Vercel nameservers on our end and get errors.

Does Vercel DNS support ".no" domains?

Thanks!

### Steps to Reproduce

When we try to set the ns1 and ns2 Vercel nameservers on our end we get these errors:
`Host [ns2.vercel-dns.com] not found in the registry.`
`Host [ns1.vercel-dns.com] not found in the registry.`

**Comment by amyegan** [comment, marked as answer]
Hi @Thuneer. Are those error messages coming from your registrar? 

Some DNS providers need you to include a `.` at the end of the domain to indicate a [fully qualified domain name](http://en.wikipedia.org/wiki/Fully_qualified_domain_name). So it would look like `ns1.vercel-dns.com.` instead of `ns1.vercel-dns.com` when you set the nameserver.

If that doesn't work, you can add some DNS records to use wildcard domains without switching to Vercel nameservers. If you would like to explore that option, you can find more info at https://vercel.com/guides/wildcard-domain-without-vercel-nameservers

Please let me know if either of those solutions work for you 🙏
